packages/typescript-typings/types/ethers/index.d.ts

```diff
@@ -31,4 +31,7 @@ declare module 'ethers' {
         public static getDeployTransaction(bytecode: string, abi: any, ...args: any[]): Partial<TxData>;
         constructor(address: string, abi: any, provider: any);
     }
+    const enum errors {
```

Comment:
Convention: Enum variable name should start with capital letter.

Reply:
This is a definition file for an existing library (ethers). My experience with TypeScript is limited, but my understanding is that I'm just defining the type for this existing member and I cannot change the name or the casing.

Reply:
Ah yep, that is correct!